serverlessworkflow/modules/ROOT/pages/use-cases/advanced-developer-use-cases/deployments/deploying-sonataflow-management-console-on-openshift.adoc

```diff
@@ -0,0 +1,149 @@
+= Deploying {product_name} Management Console on OpenShift
+:compat-mode!:
+// Metadata:
+:description: Deploying Management Console on OpenShift
```

Comment:
Can we make `Deploying Management Console on Kubernetes`?

We replaced `oc` with `kubectl`, and in the routes section, we added a note that's only on OCP. For Kubernetes, users can create an Ingress: https://kubernetes.io/docs/concepts/services-networking/ingress/